bin/markup.pl

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+use FindBin;
+use lib $FindBin::Dir . "/../lib";
+my $root = $FindBin::Dir . "/../";
+
+use autodie;
+use File::Copy;
+use File::Spec::Functions;
+use File::Path qw/ make_path /;
+use JSON::MaybeXS;
+
+my $build_dir = catdir( $ENV{HOME}, 'ddgc/ddh-userpages' );
+
+sub json {
+    local $/;
+    open my $fh, '<:encoding(UTF-8)', catfile( $build_dir, 'users.json' );
+    return <$fh>;
+}
+
+my $users = JSON::MaybeXS->new( utf8 => 1 )->decode( json );
+for my $user ( @{ $users } ) {
+    my $user_dir = catdir( $build_dir, $user );
+    make_path( $user_dir );
+    my $target = catfile( $user_dir, 'index.html' );
+    unlink $target;
```

Comment:
Thanks @MariagraziaAlastra - I've wrapped this in a check to ensure the file exists before nuking it.

We have to do it this way with the delete first since:

- There are existing index.html files generated from templates which need to be overwritten.
- There's no option in perl `symlink` (that I know of) to "force" link generation.